### ISSUE TITLE
Manage toolbar position

### ISSUE THREAD

**Issue by tumpio** [tumpio/searchbuttonsbar#3]
Feature request taken from user review:
> Giving the user the option to put the bar above the browser or on the bottom with the status bar would also be nice (as I've kept my search window on my status bar for years).

https://addons.mozilla.org/en-US/firefox/addon/searchbuttonsbar/reviews/840766

**Comment by tumpio** [comment]
Two possible solutions for this already exist:

User can change the position of the toolbar by using my other add-on called Toolbar Position Changer:
https://addons.mozilla.org/addon/toolbar-position-changer

User can also move the Search Buttons to other toolbar in Firefox Customize view by dragging it from toolbar to toolbar.

**Comment by Antipodus** [comment]
Thanks for the pointer for the Toolbar Position Changer.  I have no idea how I missed an extension that on target for my needs.  It would have saved me a bit of trouble, as I've already used the drag-&-drop method and a few other tricks to get the layout I like.

I believe that the Manage Search Engines extension and the existing search window pop-up of loaded search engines would suffice for those who have a lengthy list of engines. 

I'd think that a button to toggle the Search Bar itself on and off the screen entirely would be the most user friendly.  The Groowe Toolbar (my current go-to) has such a button so that the bar itself is on my screen only when needed and tucked away the rest of the time.

I'm also currently using a toggle button via the Custom Buttons extension to toggle my search window & a collection of other tools that I call upon infrequently -- just above my status bar.

While I'm not doing a lot of off the wall searches these days, it's clear to me that the ease of access from your bar would work really well for librarians, researchers, and the like.  The Groowe is O.K. for general (admittedly pedestrian) use, but quick access to a one click fast access list of specialized engines seems tailor made for academics.    

Postscript:  Just saw the note about Firefox 57, and all I can say is Damn!  Extensions are what make Firefox worth the trouble... I may end up using Palemoon again (or even Slimjet) if things pan out too badly.

**Comment by Eleventythreeve** [comment]
Is there a PM version of TBC? When we checked on AMO it said that it had been removed.

Our reasoning is we want to replace an old abandoned(and kinda questionable) search extension but have one that organizes search engines instead of the plain list.

**Comment by tumpio** [comment]
> Is there a PM version of TBC? When we checked on AMO it said that it had been removed.

No, some fixes would be required to make it PM compatible. See https://github.com/tumpio/toolbarpositionchanger/issues/12

All legacy add-ons were disabled from AMO. They are still accessible for developers. Sources can be found from https://github.com/tumpio/toolbarpositionchanger if some one wants to port it to PM.

**Comment by Eleventythreeve** [comment]
Ultimately we dropped the old extension and TBC wouldn't be relevant since we don't use a toolbar for search engines. 

Still, for some people it might have value. Maybe you should port your extensions over to PM. They badly need developers. 

Right now it is the only real open source browser out there. Everyone knows that the personalities of it's developers are horrific but open source browsing needs to survive. 

Maybe in your own way you could add to the mix.
